src/pages/Home.css

```diff
@@ -1,30 +1,32 @@
 #root {
-  background-color: rgb(190 221 211);
+  background-color: #3685df;
   height: 100vh;
   overflow-x: hidden;
   font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  color: white;
 }
 
 .quran-details {
   padding: 7%;
 }
 
 .heading {
-  background-color: rgb(140 205 186);
+  background: #1f5ca3;
 }
 
 #myDiv .col {
-  box-shadow: 2px 2px 2px  lightblue;
+  background: rgba(60, 145, 233, 0);
+  box-shadow: 2px 2px 2px darkblue;
 }
```

Comment:
## Your Design
![image](https://user-images.githubusercontent.com/93039359/226111128-76c4ca75-a335-4eb6-a2c7-9096f7268f83.png)

## Cindy's Design
![image](https://user-images.githubusercontent.com/93039359/226111221-5bc7eba9-d036-4fd2-974f-a6558da60f59.png)

- [ ] Great work on implementing the design but it would be nice if you make your cards dark to make your design background look similar to Cindy's design as suggested by the previous reviewer :blush: